pkg/api/kyverno/v1/policy_types.go

```diff
@@ -240,6 +257,16 @@ type ExcludeResources struct {
 	ResourceDescription `json:"resources,omitempty" yaml:"resources,omitempty"`
 }
 
+// AnyAll allow users to "AND" or "OR" between resources
+type AnyAll []struct {
```

Comment:
Can we name it precisely what this object contains? For example, `resourceFilters`?